src/main/java/org/asamk/signal/json/JsonGroupInfo.java

```diff
@@ -32,6 +32,7 @@
 
     JsonGroupInfo(SignalServiceGroupV2 groupInfo) {
         this.groupId = Base64.encodeBytes(GroupUtils.getGroupId(groupInfo.getMasterKey()));
+        // TODO populate members and name fields
```

Comment:
Looking at the Signal-Android code for the `SignalServiceGroupV2` class, I'm not experienced enough to see how we'd get the name/members out of it. I'll remove the comment for now, and either way #388 exists so this can be pursued or "wont-fix" later.